backend/src/main/java/com/bulletjournal/repository/models/User.java

```diff
@@ -21,6 +21,9 @@
     @Column(nullable = false)
     private Integer points = 0;
 
+    @Column(nullable = false, name = "user_timestamps")
+    private String userTimestamps;
```

Comment:
We instead this to be a json blob, so in the future we can have different timestamps for other case